<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ServiceDialogUI</class>
 <widget class="QDialog" name="ServiceDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Modify Services</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QTreeView" name="mServiceView" />
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="mNewServiceButton" >
         <property name="text" >
          <string>&amp;New Service</string>
         </property>
         <property name="shortcut" >
          <string>Alt+N</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mDeleteServiceButton" >
         <property name="text" >
          <string>&amp;Delete Service</string>
         </property>
         <property name="shortcut" >
          <string>Alt+D</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>101</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOKButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>mOKButton</sender>
   <signal>clicked()</signal>
   <receiver>ServiceDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel" >
     <x>246</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ServiceDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>397</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel" >
     <x>397</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
